### ISSUE TITLE
redirect_to with the generated JWT token

### ISSUE THREAD

**Issue by 5hirish** [singingwolfboy/flask-dance#172]
I want to redirect to the URL that returns the generated JWT Token. But the `redirect_to` argument is set early before the OAuth dance starts and the JWT token is generated after the OAuth dance completes. So, how do I return the generated JWT token after the OAuth dance completes?

```
twitter_blueprint = make_twitter_blueprint()
...
twitter_blueprint.redirect_to = "auth.get_jwt_token"
... If not logged in:
redirect(url_for("twitter.login"))
... Generate JWT Token
```

**Comment by 5hirish** [comment]
So, what I am doing now is maintaining a flask session and setting the generated JWT token when OAuth dance completes and the send this JWT token from session via `redirect_to` route and clear the session. But I don't want to maintain this session from server app. What I expect is to:
```
1) Setup Twitter Blueprint => 
2) Redirect to Twitter login => 
3) Login Success => 
4) Get AccessToken => 
5) Generate JWT Token => 
6) Setup redirection to the twitter blueprint with JWT Token as Header => 
7) Redirect and return JWT Token
```

But since I have to setup the redirection arround step 1, I can't set the JWT Token there.

Back to your suggestion, I did not understand how chain of redirects would help me, or I did not understand what a chain of redirection is.
